.github/workflows/main.yml

```diff
@@ -78,15 +83,19 @@ jobs:
         - target: i686-pc-windows-msvc
           os: windows-latest
           rust: nightly
+          test_verbatim: 1
         - target: x86_64-pc-windows-msvc
           os: windows-latest
           rust: nightly
+          test_verbatim: 1
         - target: i686-pc-windows-gnu
           os: windows-latest
           rust: nightly-i686-gnu
+          test_verbatim: 1
         - target: x86_64-pc-windows-gnu
           os: windows-latest
           rust: nightly-x86_64-gnu
+          test_verbatim: 1
```

Comment:
It's fine to not test this on -gnu targets, fundamentally it seems to be a limitation of the underlying toolchain.